ocaml/xapi/cli_operations.ml

```diff
@@ -3821,6 +3827,9 @@ let vm_export_aux obj_type fd printer rpc session_id params =
   let num = ref 1 in
   let uuid = List.assoc (obj_type ^ "-uuid") params in
   let ref = Client.VM.get_by_uuid rpc session_id uuid in
+  if obj_type = "snapshot" then
```

Comment:
This could be simplified to 
```
if obj_type = "snapshot" && not (Client.VM.get_is_a_snapshot rpc session_id ref) then
  failwith (Printf.sprintf ..);
```